package.json

```diff
@@ -15,18 +15,27 @@
   },
   "scripts": {
     "cleanup": "./cleanup.sh",
-    "build": "yarn workspaces foreach run build",
+    "build": "yarn workspaces run build",
     "watch": "tsc --build --watch packages/*/src",
-    "test": "yarn workspaces foreach run test",
-    "lint": "yarn workspaces foreach -p run lint",
-    "lint:fix": "yarn workspaces foreach -p run lint:fix"
+    "test": "yarn workspaces run test",
+    "lint": "yarn workspaces -p run lint",
+    "lint:fix": "yarn workspaces -p run lint:fix",
+    "coverage": "yarn workspaces -p run coverage"
   },
   "devDependencies": {
+    "@types/chai": "^4.2.0",
     "@types/mocha": "^7.0.2",
     "@types/node": "^14.0.13",
     "chai": "^4.2.0",
     "eslint-plugin-simple-import-sort": "^5.0.3",
     "mocha": "^8.0.1",
+    "nyc": "^14.1.0",
+    "prettier": "2.0.5",
+    "ts-node": "^8.1.0",
+    "tslint": "^5.16.0",
+    "tslint-config-prettier": "^1.18.0",
+    "tslint-consistent-codestyle": "^1.15.1",
+    "tslint-plugin-prettier": "^2.0.1",
```

Comment:
we don't need `prettier`

Also, later on we will need to remove `tslint`

Reply:
Removed